cost/aws/unused_rds/unused_rds.pt

```diff
@@ -7,7 +7,7 @@ severity "low"
 category "Cost"
 default_frequency "weekly"
 info(
-  version: "6.1",
+  version: "6.2",
```

Comment:
We don't really need to bump the policy template version if it's only a README update fiw

Reply:
```suggestion
  version: "6.1",
```

Reply:
Thanks for the info, I'll update it